<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startMainPolygonOuterRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始绘制主多边形外环</string>
    </property>
   </widget>
   <widget class="Canvas" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startMainPolygonInnerRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始绘制主多边形内环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTailorPolygonOuterRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始绘制裁剪多边形外环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTailorPolygonInnerRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始绘制裁剪多边形内环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>裁剪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishMainPolygonOuterRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>完成绘制主多边形外环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishMainPolygonInnerRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>完成绘制主多边形内环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishTailorPolygonOuterRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>完成绘制裁剪多边形外环</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishTailorPolygonInnerRingBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>完成绘制裁剪多边形内环</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header location="global">canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
